.github/workflows/test-js-packages.yml

```diff
@@ -509,6 +509,59 @@ jobs:
         working-directory: ./examples/codegen_verifier
         run: ./test.sh
 
+  external-repo-checks:
+    needs: [build-nargo]
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        project:
+         - { repo: AztecProtocol/aztec-nr, path: ./ }
+         - { repo: AztecProtocol/aztec-packages, path: ./noir-projects/noir-contracts }
+         # Disabled as aztec-packages requires a setup-step in order to generate a `Nargo.toml`
+         #- { repo: AztecProtocol/aztec-packages, path: ./noir-projects/noir-protocol-circuits }
+         - { repo: zac-williamson/noir-edwards, path: ./, ref: 0016ce82cd58b6ebb0c43c271725590bcff4e755  }
+        # TODO: Enable these once they're passing against master again.
+        #  - { repo: zac-williamson/noir-bignum, path: ./, ref: 030c2acce1e6b97c44a3bbbf3429ed96f20d72d3  }
```

Comment:
Looks like there were 2 commits since 030c2a which have since fixed the errors. (why specify ref here instead of branch? I'd think we want to test the most recent version instead of a ref which will only get more outdated)

Reply:
> why specify ref here instead of branch?

A couple of reasons:
- We don't want our CI to fail because someone has pushed a broken commit to their master branch.
- concerns about external developers potentially being able to get us to run code contained in their repo in our CI and do bad things.

We can then have a recent fixed commit for consistency and we can update when necessary.